VERSION

```diff
@@ -1 +1 @@
-1.33.0
+1.33.0.p
```

Comment:
Perhaps use -p0 here to keep with semver spec.  We can increment the patch version that way but still keep tied to the upstream version.

Reply:
Ha, I really meant it as "Papyrus" (i.e., Papyrus changes on top of upstream 1.33.0), but you're right, should version the patch as well. Didn't want to use the actual patch digit since that's used by the upstream. The official method is `<major>.<minor>.<patch>-<pre-release>+<build>`, so I'm thinking `1.33.0-papyrus+001` (with the +001 being the increment for the Papyrus additions).

I think it'll also be good to also maintain a `latest` tag in the same way that Amazon ECS is doing it, so we don't have to update the LT/CE every single time we want to update the ECS agent version (seems to be changing fairly often upstream).

Reply:
I'm fine with that version nomenclature.

Using a rolling tag would mean we can't do immutable tags...  Definitely a balance there to strike but if you'd prefer to do so in this case, can we please use something other than 'latest'?  It's far too easy to accidentally move 'latest' since it's the default in the client.  Perhaps have a rolling 'prod', 'stg', and 'dev' tag that appropriate environments can be configured for.

Reply:
Yeah, I did want to use immutable tags, but we can reinstate that in the future (download "latest" tarball from S3 bucket and skip ECR entirely, see other thread re: init).

I was thinking of using `latest` within each environment for now (while we have separate ECR per account, so `latest` is really the latest in that environment), and once we consolidate the ECRs w/ cross-account ECR roles then we can switch to env-specific tags for that, WDYT?

Reply:
Something of a judgment call.  I would probably tag it differently now so all we have to do later is update the repo but I'm good either way.